api/vendor/github.com/openshift/assisted-service/models/v2_cluster_update_params.go

```diff
@@ -273,6 +282,22 @@ func (m *V2ClusterUpdateParams) validateClusterNetworks(formats strfmt.Registry)
 	return nil
 }
 
+func (m *V2ClusterUpdateParams) validateControlPlaneCount(formats strfmt.Registry) error {
+	if swag.IsZero(m.ControlPlaneCount) { // not required
+		return nil
+	}
+
+	if err := validate.MinimumInt("control_plane_count", "body", *m.ControlPlaneCount, 3, false); err != nil {
+		return err
+	}
+
+	if err := validate.MaximumInt("control_plane_count", "body", *m.ControlPlaneCount, 5, false); err != nil {
+		return err
+	}
+
+	return nil
+}
```

Comment:
What about if it's a SNO cluster? Couldn't the ControlPlaneCount be 1?

Reply:
Yes this is a mistake I forgot to change it the update params as well, thanks